app/models/cycle.rb

```diff
@@ -0,0 +1,3 @@
+class Cycle < ActiveRecord::Base
+  has_many :paws_up
```

Comment:
you are not using this accessor method but I think it will only associate correctly if you do

```ruby
has_many :paws_ups
```